.gitignore

```diff
@@ -15,7 +15,6 @@ gradle-app.setting
 # gradle/wrapper/gradle-wrapper.properties
 
 HELP.md
-.gradle
```

Comment:
I think this should stay here.
`.gradle` is a special folder and can exist under certain conditions (e.g. when `GRADLE_HOME` is defined at the workspace root).
It is not our use case but is a nice safety belt to keep.

Reply:
I fully agree but that's already declared at the first line of this file :)

Reply:
hmm, I missed this, sorry.
After reading the file, I may have kept the `.gradle` you deleted and removed the 4 first lines of the file.
But we can totally rework `.gitignore` accross the modules at a later date if needed.

Reply:
If you do have any idea regarding how you'd want to rework them, that'd be a pleasure :)